<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AResultsPageClass</class>
 <widget class="QWidget" name="AResultsPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="upRefreshPushButton">
       <property name="text">
        <string>&amp;Get Results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upScorePushButton">
       <property name="text">
        <string>&amp;Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upEstimatePushButton">
       <property name="text">
        <string>&amp;Estimate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Solutions summary</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="upSolutionsTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Solution details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="upSolutionTexEdit"/>
      </item>
      <item>
       <widget class="QTableWidget" name="upSolutionStatisticsTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>upRefreshPushButton</tabstop>
  <tabstop>upScorePushButton</tabstop>
  <tabstop>upEstimatePushButton</tabstop>
  <tabstop>upSolutionsTableWidget</tabstop>
  <tabstop>upSolutionTexEdit</tabstop>
  <tabstop>upSolutionStatisticsTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
